sambacc/commands/check.py

```diff
@@ -34,7 +36,7 @@ def check(ctx: Context) -> None:
     """Check that a given subsystem is functioning."""
     if ctx.cli.target == "winbind":
         cmd = samba_cmds.wbinfo["--ping"]
-        samba_cmds.execute(cmd)
+        os.execvp(cmd.name, cmd.argv())
```

Comment:
There's already a execvp wrapper in the samba_cmds.py file: https://github.com/samba-in-kubernetes/sambacc/blob/b6e11822764dc7e31472d0f93910ab42417ac4f8/sambacc/samba_cmds.py#L212
Is there a reason you didn't use it?

Reply:
[Game show host voice] So close, but I'm afraid the answer is CommandArgs. The judges would also have accepted a new protocol, but typing.Any is too broad. [/joke] :-)

CommandArgs is the right choice because it is the base class of SambaCommand, thus all SambaCommands are CommandArgs. Annotating it with typing.Any is too general and would have the type checker alloing things like `execute(7)` which would fail at runtime because an int doesn't have a .name or .argv method.

Reply:
Ah..I'm totally exposed now, just kidding.

Updated.